test/Spec.hs

```diff
@@ -19,7 +19,7 @@ import qualified Database.PostgreSQL.Simple.FromRow as Sql
 
 
 connectionSettings :: ByteString
-connectionSettings = "host=localhost port=5432 user=postgres dbname=pg_named"
+connectionSettings = "host=localhost port=5432 user=postgres password=helloci dbname=pg_named"
```

Comment:
[POSTGRES_PASSWORD](https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_password) must be provided in recent versions of postgres image, so I had to add it (also in CI and readme)